bib/cmd/bootc-image-builder/image.go

```diff
@@ -77,20 +75,74 @@ func Manifest(c *ManifestConfig) (*manifest.Manifest, error) {
 	}
 }
 
-func applyFilesystemCustomizations(customizations *blueprint.Customizations, c *ManifestConfig) error {
-	// Check the filesystem customizations against the policy and set
-	// if user configured
-	if err := blueprint.CheckMountpointsPolicy(customizations.GetFilesystems(), policies.OstreeMountpointPolicies); err != nil {
-		return err
-	}
-	if customFS := customizations.GetFilesystems(); len(customFS) != 0 {
-		allowedMounts := []string{"/", "/boot"}
-		for _, mp := range customFS {
-			if !slices.Contains(allowedMounts, mp.Mountpoint) {
-				return fmt.Errorf("cannot use custom mount points yet, found: %v", mp.Mountpoint)
-			}
+var (
+	// The mountpoint policy for bootc images is more restrictive than the
+	// ostree mountpoint policy defined in osbuild/images. It only allows /
+	// (for sizing the root partition) and custom mountpoints under /var but
+	// not /var itself.
+
+	// Since our policy library doesn't support denying a path while allowing
+	// its subpaths (only the opposite), we augment the standard policy check
+	// with a simple search through the custom mountpoints to deny /var
+	// specifically.
+	mountpointPolicy = pathpolicy.NewPathPolicies(map[string]pathpolicy.PathPolicy{
+		// allow all existing mountpoints to support size customizations
```

Comment:
Maybe: `// allow all existing mountpoints (but no subdirs) to support size customizations`